packages/vuepress-theme-odyssey/styles/docskit/_Visual.scss

```diff
@@ -0,0 +1,43 @@
+.docskit-visual {
+  grid-column: 2 / 3;
+  margin-bottom: $spacing-m;
+
+  &,
+  p {
+    font-size: ms(0);
+  }
+
+  &.docskit-visual--wide {
+    grid-column: 1 / 3;
+    width: 100%;
+  }
+
+  .docskit-visual--title {
+    display: flex;
+    align-items: center;
+    font-weight: normal;
+
+    &::before {
+      margin-right: $spacing-xs;
+      font-size: $size-title-5;
+    }
+  }
+
+  &.is-docskit-visual-positive .docskit-visual--title::before {
+    content: '🛎️';
+  }
+
+  &.is-docskit-visual-negative .docskit-visual--title::before {
+    content: '🚫';
+  }
```

Comment:
Can these be in markup as a component?

Reply:
I'm not strongly opinionated here.

I know these were chosen specifically to avoid needing translation, which means they're safe as `content`. It's nice to keep something that feels like semantic decoration within the presentation layer.

I don't think we'd want this to be a open slot or prop since the use case is particularly specific, but I'm not opposed to moving this elsewhere if there are benefits.